ports/raspberrypi/boards/maple_elite_pi/pins.c

```diff
@@ -0,0 +1,53 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    { MP_ROM_QSTR(MP_QSTR_GP0), MP_ROM_PTR(&pin_GPIO0) },
```

Comment:
The board's silk has legends like "D5", "A0", etc. This file should reflect the markings on the silk, when it makes sense to do so.

Reply:
While that makes sense for primaries, should I also include the aliases in the pinout diagram? https://d33wubrfki0l68.cloudfront.net/d42e46a6982528aa7328b87dc758414ebabc5dbf/7d77f/assets/images/elite-pi_pinout-both-3c2ffb9c7e7d46f36fd54006ab000709.png

Reply:
Taking this pin for example, you might define both D29 and A3 but not any of SCL0, RX0, /CS1, or 6B.

![image](https://user-images.githubusercontent.com/1517291/198651830-061895a6-a556-490f-b967-838280a0ee09.png)

When there's e.g., a single set of pins designated for SPI, UART, or I2C we name them separately. So, for instance, if the use of the pins labeled D0/D1 as TX/RX is well established for this form factor, I might put that alias even though it's not on the silkscreen. Similarly, D23/D20/D22 might be SCK/MISO/MOSI.  I'm basing both of these on what was done on the Adafruit KB2040, which designates the pins in those locations for those purposes.

On the other hand, you can just concentrate on getting the basic "matches the silk" done, and someone can later add an alias. Removing an alias, even if mistaken, is tougher because it creates an incompatible change and we don't make incompatible changes lightly.

Reply:
I'll just use the non-duplicated names for now.